src/components/ArticlePage/MarkdownHtml.js

```diff
@@ -89,8 +89,15 @@ const Html = styled.div`
   }
   h2 {
     ${H4.styles}
-    margin: ${2 * baselineSpacing}px 0 ${baselineSpacing}px 0;
+    margin: ${1.5 * baselineSpacing}px 0 ${baselineSpacing}px 0;
     max-width: ${props => props.theme.contentMaxWidth}px;
+    border-top: 1px solid #CCCCCC;
+    padding-top: 29px;
```

Comment:
This value is quite magical. You could write it in a more understandable way:

```suggestion
    padding-top: ${5 * baselineSmall - 1}px; // padding - border width
```